tortoise/verifying_tortoise_test.go

```diff
@@ -3074,12 +3074,39 @@ func TestRerunMissingLayer(t *testing.T) {
 	}
 }
 
-func BenchmarkTortoiseLayerHandling(b *testing.B) {
-	ctx := context.Background()
+func TestOnlyInorderLayersAreVerified(t *testing.T) {
 	s := sim.New(sim.WithLayerSize(defaultTestLayerSize))
 	s.Setup()
 
+	ctx := context.Background()
+	cfg := defaultConfig(t, s.State.MeshDB, s.State.AtxDB)
+	tortoise := NewVerifyingTortoise(ctx, cfg)
+
+	var (
+		inorder  types.LayerID
+		verified types.LayerID
+	)
+	for _, lid := range sim.GenLayers(s,
+		sim.WithSequence(1),
+		sim.WithSequence(1, sim.WithNextReorder(1)),
```

Comment:
can you add comments about what `WithNextReorder()` do or give a concrete example for the layers generated by these sequences? i find this hard to understand quickly.

Reply:
in this case the layers are 8, 10, 9, 11, 12 . i will try to clarify that in the WithNextReorder, it basically defines the delay for inorder layer